README.Rmd

```diff
@@ -15,14 +15,25 @@ knitr::opts_chunk$set(
 # r2dii.banks <a href='https://github.com/2DegreesInvesting/r2dii.banks'><img src='https://imgur.com/A5ASZPE.png' align='right' height='43' /></a>
 
 <!-- badges: start -->
-[![lifecycle](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://www.tidyverse.org/lifecycle/#experimental)
 [![CRAN status](https://www.r-pkg.org/badges/version/r2dii.banks)](https://CRAN.R-project.org/package=r2dii.banks)
-[![Launch Rstudio Binder](http://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/maurolepore/r2dii.banks/master?urlpath=rstudio)
 <!-- badges: end -->
 
-r2dii.banks is a collection of R packages that 2DII developed for banks.
+The think tank '2DII' (2 Degrees Investing
+Initiative; <https://2degrees-investing.org/>) maintains a set of
+packages designed for banks. Together, those packages implement in R a
+methodology known as 'PACTA' (Paris Agreement Capital Transition
+Assessment), which enables users to measure the alignment of financial
+portfolios with climate goals. This package makes it easy to install
+and load all '2DII' packages for banks in a single step.
 
-Install r2dii.banks with:
+## Installation
```

Comment:
This is to make this package consistent with all other packages. The header also allows us to link the installation section from anywhere.